src/limbo/opt/parallel_repeater.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef LIMBO_OPT_PARALLEL_REPEATER_HPP
+#define LIMBO_OPT_PARALLEL_REPEATER_HPP
+
+#include <algorithm>
+
+#include <boost/math/special_functions/sign.hpp>
+
+#include <Eigen/Core>
+
+#include <limbo/tools/parallel.hpp>
+
+namespace limbo {
+    namespace defaults {
+        struct parallel_repeater {
+            BO_PARAM(int, repeats, 10);
+        };
+    }
+    namespace opt {
+        template <typename Params, typename Optimizer>
+        struct ParallelRepeater {
+            template <typename F>
+            Eigen::VectorXd operator()(F&& f)
+            {
+                tools::par::init();
+                typedef std::pair<Eigen::VectorXd, double> pair_t;
+                auto body = [&](int i) {
+                // clang-format off
+                // we need a copy because each thread should touch a copy of the struct!
+                F f_copy = f;
```

Comment:
You ment inside the lambdas? We turned off clang inside them because it doesn't work properly with them. You think we should add indentation inside them also?